pandas/tests/io/msgpack/test_pack.py

```diff
@@ -25,6 +26,7 @@ def testPack(self):
             (), ((),), ((), None,),
             {None: 0},
             (1 << 23),
```

Comment:
can you add a round-trip test for the example in the issue itself.

Reply:
This test should already test for the buggy behavior when it comes to `np.bool_`.

Should I revert the changes to this test and add a one checking round-trip behavior or should I make that test a additional one? 
Where would I best add this test? (There is no /test/io/formats/test_to_msgpack.py)